tests/integration/Router/RouterTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Baka\Test\Integration\Router;
+
+use Baka\Router\Route;
+use PhalconUnitTestCase;
+
+class RouterTest extends PhalconUnitTestCase
+{
+    public function testGetRoute()
+    {
+        $router = Route::get('/')->controller('TestController');
+
+        $this->assertTrue($router instanceof Route);
+        $this->assertTrue(in_array('get', $router->getVia()));
+        $this->assertTrue($router->getController() === 'TestController');
```

Comment:
We can use `assertEquals` instead of `assertTrue`. Notice the order of the parameters on https://codeception.com/docs/modules/Asserts#assertEquals , the expected should be the first param.